src/quo/components/inputs/address_input/view.cljs

```diff
@@ -108,14 +109,14 @@
                          (when-not (empty? scanned-value)
                            (on-change scanned-value)))
                        [scanned-value])
-        [rn/view {:style style/container}
+        [rn/view {:style (style/container container-style)}
          [rn/text-input
           {:accessibility-label    :address-text-input
            :style                  (style/input-text theme)
            :placeholder            (i18n/label :t/name-ens-or-address)
            :placeholder-text-color placeholder-text-color
            :default-value          @value
-           :auto-complete          (when platform/ios? :none)
+           :auto-complete          (when platform/ios? :off)
```

Comment:
Since the `address-input` is being used by the `page-top` component, I had to fix this, it throws a warning that breaks the tests